### PR TITLE
Editing Pass

### PR DESCRIPTION
**Pull request by lmilechin**
Doing an editing pass before releasing the site.

### CODE REVIEW

File: docs/orcd-systems.md [hunk 1]
```diff
@@ -41,41 +44,41 @@ Satori is an IBM Power 9 large memory node system. It is open to everyone on cam
...
 ### Satori Quick Links
 
-- Documentation: [(https://mit-satori.github.io/](https://mit-satori.github.io/)
+- Documentation: [https://mit-satori.github.io/](https://mit-satori.github.io/)
 - OnDemand web portal: [https://satori-portal.mit.edu](https://satori-portal.mit.edu)
 - Help: Send email to [orcd-help-satori@mit.edu](mailto:orcd-help-satori@mit.edu)
 - Slack: [https://mit-satori.slack.com/](https://mit-satori.slack.com/)
 
+<!--
+Do we want to keep the link to the Satori slack?
```
Comment (lmilechin) [inline docs/orcd-systems.md:65]: Remove this link

**Comment by shaohao99** [comment]
One change. Remove the "Lustre Scratch" row in the table of the file system page. It is completely retired now.

File: docs/filesystems-file-transfer/filesystems.md [hunk 2]
```diff
@@ -43,4 +47,4 @@ OpenMind provides a number of different storage options. See the [OpenMind Docum
 | Vast Scratch | `/om2/scratch/<week-day>` | N/A | Not backed up <br> purged 2 weeks after creation | Scratch space |
 | Lustre Scratch | `/nobackup/scratch/<week-day>` | N/A | Not backed up <br> purged 3 weeks after creation | Scratch space <br>  See [Lustre Best Practices](https://www.nas.nasa.gov/hecc/support/kb/lustre-best-practices_226.html) page |
```
Comment (shaohao99) [inline docs/filesystems-file-transfer/filesystems.md:48]: Remove the "Lustre Scratch" row in the table of the file system page. It is completely retired now.

**Review by christophernhill — APPROVED** [review]
@lmilechin  - this looks good to me. Let's go for it.

**Pull request merged**